src/route/RouteData.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {BrowserRouter as Router, Route, Routes, Link} from 'react-router-dom';
+// import Login from '../Pages/Auth/Login';
+// import MockAPI from '../Helpers/Helpers/MockMan';
+import MainPage from '../pages/MainPage';
+// import MainPage from '../pages/MainPage/index';
+// import Dashboard from '../Pages/Dashboard/index';
+
+const LoginPage = React.lazy(() => import('../pages/Auth/Login'));
+const SignUpPage = React.lazy(() => import('../pages/Auth/SignUp'));
+
+const NoMatch = () => {
+    return (
+      <div>
+        <h2>404.. This page is not found!</h2>
+        <p>
+          <Link to="/">Go to the home page</Link>
+        </p>
+      </div>
+    );
+  }
+
+
+//Routing Data
+const RouteData = () =>{
+    return(
+        <Routes>         
+            <Route exact path='/signup' element={<SignUpPage/>}/> 
+            <Route exact path='/login' element={<LoginPage/>}/> 
+            {/* <Route exact path='/mockman' element={<MockAPI/>}/> */}
+            <Route path='/*' element={<MainPage/>}/>
+        </Routes>
+   );
+}
+export default RouteData;
```

Comment:
I'm assuming this route is meant for authentication since a RouteData file already exists in the MainPage. If so, I would suggest you rename the file to AuthRoute and the previous to AppRoute

Reply:
ok